code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -162,7 +162,6 @@
 	item_state = "cardborg"
 	body_parts_covered = CHEST|GROIN
 	flags_inv = HIDEJUMPSUIT
-	dog_fashion = /datum/dog_fashion/back

```

Comment:
Probably didn't want to remove these ? Something gone wrong with the merge probably.